backend/src/test/java/de/neuefische/backend/controller/PositionsControllerTest.java

```diff
@@ -172,7 +192,7 @@ void putPositionById_unsuccessful() {
                 .headers(http -> http.setBearerAuth(jwt1))
                 .bodyValue(updatedPosition)
                 .exchange()
-                .expectStatus().is5xxServerError(); //needs to be refactored after adding appropriate controller advice
+                .expectStatus().isNotFound(); //needs to be refactored after adding appropriate controller advice
```

Comment:
Ist der Kommentar in dieser Zeile trotz der Code-Änderung noch aktuell?

Reply:
Oh, tatsächlich nun nicht mehr. Gut gesehen, danke!